### DISCUSSION TITLE
MQTT to HA

### DISCUSSION THREAD

**Discussion by figarospares** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#689] [Q&A]
Hi guys I have got this setup and UI work perfect I have tried for over 2 hours to get this to work in HA but no sucsess I am using HA in Docker on Synology with Zigbee2MQTT in docker as well and using Mosquito Eclipse in Synology as Broker I have over 30 devices on zigbee working fine but I can not see to add this. Below is config on zigbee2MQTT and pic of MQTT ui on hot tub any help would be great...

homeassistant: true
frontend: true
permit_join: true
mqtt:
  base_topic: zigbee2mqtt
  server: mqtt://127.0.0.1:1883
  keepalive: 60
  reject_unauthorized: false
  version: 4
  password: west
  user: hottub
  client_id: lazyspa

![Screenshot (116)](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/109300373/a57ef42f-5db6-4838-ad0a-62128fcc6bcb)

**Comment by visualapproach** [comment]
It takes 2-3 minutes after restart until ESP connects to MQTT. What's the MQTT status on the main UI page (a few minutes after restart)?

**Reply by figarospares** [reply]
MQTT: CONNECT_FAILED

**Reply by visualapproach** [reply]
OK so it can't connect to the broker. I don't know how mosquitto eclipse works, but check that they are on the same wifi network and that the broker and ESP has the same settings for username and pswd. Check the logs in the broker for any clues.

**Comment by chunkysteveo** [comment]
Your mqtt ip address cannot be 127.0.0.1.... it needs to be the ip address where the mqtt broker sits on your network.

**Comment by figarospares** [comment]
All is working now I re-installed mosquitto and now all on long thanks for the help guys
